.circleci/config.yml

```
@@ -46,6 +46,22 @@ commands:
           root: .
           paths:
             - ./pkg/
+  build-and-persist-packer-binary-arm64:
+    parameters:
+      GOOS:
+        type: string
+      GOARCH:
+        default: "arm64"
```

I think it would be a good idea to have something like this instead ( well and reuse the already existing block ) :


```suggestion
  build-and-persist-packer-binary:
    parameters:
      GOOS:
        type: string
      GOARCH:
        default: "amd64"
```

This deduplicates code a little

---

👍